sandworm/lib/web.mlx

```diff
@@ -110,6 +110,15 @@ $ dune exec hello_world
     "For more information on Dune and Package Management, check the "
     <Link href="https://dune.readthedocs.io/en/latest/"> "latest dune docs." </Link>
     </p>
+    <h3 class_="text-xl font-medium pt-4 pb-2"> "Editor Setup" </h3>
+    <p>
+      "If you're using an LSP-enabled editor, you can point your LSP to run this command to start the OCaml LSP server:"
+    </p>
+    <Code>
+"
+dune tools exec ocamllsp
+"
```

Comment:
__question (non-blocking)__: Don't you think we should add example for neoVIM / Emacs / VSCode (or maybe this is for future us)?

Reply:
would we do this in a labeled pane or with `<details>`? Like:

---

<details>
  <summary>Set up for neovim</summary>
  <pre>
local lsp = require('lspconfig')
lsp["ocamllsp"].setup {
  cmd = { "dune", "tools", "exec", "ocamllsp" }
}
  </pre>
</details>
<details>
  <summary>Set up for emacs</summary>
</details>
<details>
  <summary>Set up for VS Code</summary>
</details>

---

If so let's gather from the team instructions on how to set it up and include them 🙌🏼

Reply:
Yes, I like the idea: it's non invasive! :+1: